### PR TITLE
chore(examples): Adding Serverless Example

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds the serverless example to the examples folder alongside a recipe for the docs website

## Changes

- Adds serverless example
- Adds new recipe

## Fixes

- https://github.com/kubeshop/tracetest-cloud/issues/435

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/examples-tutorials/recipes/testing-lambda-functions-with-opentelemetry-tracetest.mdx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+git clone https://github.com/kubeshop/tracetest.git
+cd tracetest/examples/quick-start-serverless
+```
+
+Before moving forward, run `npm i` in the root folder to install the dependencies.
+
+```bash
+npm i
+```
+
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/testing-lambda-functions-with-opentelemetry-tracetest.mdx:65]: The prerequisites here are quite short. Could you include more steps here? Mainly I think the ones from the readme can be put here and we can remove them from the readme and link to this recipe.

Comment (adnanrahic) [reply docs/docs/examples-tutorials/recipes/testing-lambda-functions-with-opentelemetry-tracetest.mdx:65]: I noticed you did not explain the .env file, how to use it, and how to configure it. Would be nice to have that in here somewhere too.

**Review by adnanrahic — COMMENTED** [review]
Some ideas about improvements.

**Review by jfermi — APPROVED** [review]

File: docs/docs/examples-tutorials/recipes/testing-lambda-functions-with-opentelemetry-tracetest.mdx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+:::note
+[Check out the source code on GitHub here.](https://github.com/kubeshop/tracetest/tree/main/examples/quick-start-serverless)
+:::
+
+[Tracetest](https://tracetest.io/) is a testing tool based on [OpenTelemetry](https://opentelemetry.io/) that allows you to test your distributed application. It allows you to use data from distributed traces generated by OpenTelemetry to validate and assert if your application has the desired behavior defined by your test definitions.
+
+[AWS Lambda](https://aws.amazon.com/lambda/) is a serverless, event-driven compute service that lets you run code for virtually any type of application or backend service without provisioning or managing servers. You can trigger Lambda from over 200 AWS services and software as a service (SaaS) applications, and only pay for what you use.
+
+[Th Serverless Framework](https://www.serverless.com/) is an open-source framework designed to help developers build and deploy serverless applications more efficiently. It abstracts away the underlying infrastructure management tasks, allowing developers to focus solely on writing code and defining the functions that comprise their applications.
```
Comment (adnanrahic) [inline docs/docs/examples-tutorials/recipes/testing-lambda-functions-with-opentelemetry-tracetest.mdx:28]:
```suggestion
[The Serverless Framework](https://www.serverless.com/) is an open-source framework designed to help developers build and deploy serverless applications more efficiently. It abstracts away the underlying infrastructure management tasks, allowing developers to focus solely on writing code and defining the functions that comprise their applications.
```

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**